Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM node:14.17.5-alpine3.14 as build
+FROM node:14.17.6-alpine3.13 as build
```

Comment:
Why the lower alpine version?

Reply:
For whatever unknown reason alpine 13 was at the top on dockerhub.

I guess expecting some kind of logical ordering is too much.